Library/Homebrew/cmd/home.rb

```diff
@@ -1,31 +1,38 @@
 # frozen_string_literal: true
 
+# typed: true
+
 require "cli/parser"
 require "cask/cask_loader"
 require "cask/exceptions"
+require "sorbet-runtime" if ENV["HOMEBREW_SORBET"]
 
 module Homebrew
+  extend T::Sig
+
   module_function
 
-  def home_args
-    Homebrew::CLI::Parser.new do
-      usage_banner <<~EOS
+  sig { params(blk: T.nilable(T.proc.bind(Homebrew::CLI::Parser).void)).void }
+  def home_args(&blk)
+    Homebrew::CLI::Parser.new do |p|
+      p.usage_banner <<~EOS
         `home` [<formula>]
 
         Open <formula>'s homepage in a browser, or open Homebrew's own homepage
         if no formula is provided.
       EOS
-      switch :debug
+      p.switch :debug
     end
   end
 
+  sig { void }
```

Comment:
Was this a placeholder for a more thorough signature in the future (not that I know what that looks like right now)? I think we should avoid `sig { void }` as it doesn't add anything to Sorbet's or our understanding of the code.

Reply:
EDIT: I've now [read the docs](https://sorbet.org/docs/sigs#returns--void-annotating-return-types) and understand its purpose. I mistakenly thought that given that `sig { void }` or the lack of it didn't mean any less/more numbers of errors, it wasn't needed. In fact, it's special. Would be good to say _something_ about this change in the commit message (even a link to the docs), so that other tired people like me don't think the same thing. :-)